core/Makefile

```diff
@@ -42,6 +42,10 @@ apple:
 	@cp target/universal/release/liblockbook_core.a ${libs_apple}
 
 
+.PHONY: qa
+qa:
```

Comment:
I would say move this to `dev_utlis`, but also if you're gonna automate this, you should add `touch lib.rs` otherwise someone is gonna have a bad time

Reply:
oh yeah lol I forgot to not put this here

Reply:
I've been a fan of adding really specific scripts that configure settings for me `./build_ios_core_pointed_at_qa.sh` and `./build_ios_core_pointed_at_prod.sh` for example

Reply:
Yeah it's nbd, removed